tools/amd_build/build_amd.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+import os
+import sys
+import argparse
+
+sys.path.append(os.path.realpath(os.path.join(
+    os.path.dirname(__file__),
+    os.path.pardir,
+    os.path.pardir,
+    'third_party')))
```

Comment:
This could work, though it means that this file "assumes" a certain structure in the repo, which means it's not really standalone and thus possibly brittle. I now wonder whether it could be more robust to have the path of the `hipify` module passed in via a command-line argument?

Reply:
Since this file will be moved hipify-torch repo, the above code will not be required.